src/main/java/frc/robot/RobotContainer.java

```diff
@@ -27,6 +27,7 @@ public class RobotContainer {
     public RobotContainer() {
         // Configure the trigger bindings
         configureBindings();
+        AutonomousSelector.buildSelector();
```

Comment:
you are not suppose to create objects in this block, this block is for applying commands as bindings and maybe to schedule some more non binding commands.
Consider to move it to other place or make it a variable in this class like a subsystem (robot init, or other place if you have)